broadinstitute/single_cell_portal_core
# Update metadata explainer image for AnnData (SCP-5761)

Existing image for "Classic" metadata upload is slightly incompatible with AnnData upload needs. This updated image should convey the requirements for AnnData without the classic metadata-specific requirements muddying the waters. 
![metadata-convention-explainer-anndata](https://github.com/user-attachments/assets/70ff674d-457b-42f7-b769-ed46226be1d4)

This PR also adds a visual cue that `.obsm key name` is a file attribute required for upload.

MANUAL TESTING

- Review the image above and confirm that no "classic metadata"-specific requirements are noted in the figure
- In your local instance, visit the AnnData experience and verify an asterisk is shown next to `.obsm key name` to indicate a required field (the validation is already implemented, there's just no visual cue on the form).

Known issue
- if >1 cluster file, cluster forms are collapsed BUT collapsed forms are unnamed (in Classic mode, each collapsed cluster form shows the filename and filesize as a file download link; for AnnData, the collaped cluster form should probably show the cluster name).
![Screenshot 2024-08-15 at 9 37 24 AM](https://github.com/user-attachments/assets/f90a4bf4-63b2-45e6-9f3a-736c08128241)
